### PR TITLE
Cpr molleweide

### PR DESCRIPTION
**Pull request by molleweide**

### CODE REVIEW

**Comment by molleweide** [comment]
what is your opinion on using `&&` for creating tight if checks.
if you look at this script: https://github.com/LukeSmithxyz/LARBS/blob/master/larbs.sh

**Comment by balupton** [comment]
It's common in other scripts, but is not something Dorothy does. It is obtuse, and makes modifications such as more conditions or more conditional commands more convoluted.

If the Dorothy style guide doesn't mention this, then it should be updated.

File: commands/cpr [hunk 1]
```diff
@@ -111,28 +111,14 @@ action="$(
...
-	if command-exists scp; then
-		tools+=(scp)
-	fi
-	if command-exists cp; then
-		tools+=(cp)
-	fi
+	command-exists rsync && tools+=(rsync)
+	command-exists gcp   && tools+=(gcp)
+	command-exists scp   && tools+=(scp)
+	command-exists cp    && tools+=(cp)
```
Comment (balupton) [inline commands/cpr:121]: It's common outside Dorothy, but it is not something Dorothy does. It is obtuse (it mixes and matches the roles of conditionals and actions, making groking a more involved process, especially for beginners), and makes modifications such as more conditions and more conditional commands more convoluted.

If bash ever had a linter rewriter (such as prettier.io for other ecosystems), I'd imagine such keystroke shortcut statements would get automatically converted into the expanded if statement upon save as the linter does its rewrites.

Refer to the style guide at https://github.com/bevry/dorothy/discussions/46

File: commands/cpr [hunk 2]
```diff
@@ -199,7 +185,7 @@ if test "$tool" = 'rsync'; then
 	#   --perms, -p: preserve permissions
 	#   --times, -t: preserve modification times
 	#   --group, -g: preserve group
-	#   --owner, 0o: preserve owner (super-user only)
+	#   --owner, -o: preserve owner (super-user only)
```
Comment (balupton) [inline commands/cpr:188]: 👍

**Review by balupton — CHANGES_REQUESTED** [review]
will merge typo

**Comment by balupton** [comment]
Merged the typo fix: https://github.com/bevry/dorothy/commit/c4fab2afee221dcf42b9b376ce10b44320193274

Will update the style guide to be more detailed on the `condition && action` subject.

**Pull request closed without merging**